packages/P4DPyPackage.dproj

```diff
@@ -2,13 +2,13 @@
     <PropertyGroup>
```

Comment:
We need to keep it all to Win32/Release by default.

Reply:
Oops - missed that bit - I don't use 32 bit anything (hardly anyone does in reality)

Reply:
We use Win32/Release because design-time packages are Win32 only.

Reply:
Oh, yeah - of course...

Are you supposed to install the design-times for all OSes? Sometimes things don't work if I don't (which is confusing)

Anyway - I'll be able to wipe Delphi again and try a clean system install for my tutorial soon :)

Reply:
Design-time packages are Win32 only, then we make it as default.

Reply:
if I skip other platforms from design-time packages, I can use the "Set active platform" option for project groups and the design-time packages will hold Win32 settings.

![Screen Shot 2022-08-09 at 11 29 18 1](https://user-images.githubusercontent.com/8376898/183675882-8704cd24-5d0a-4522-89c7-d7c3129bdf52.png)

Reply:
In the case of P4D Data Sciences in particular it would be far easier to manage if there was a branch that only contained design-time and another that contained run-time cos this is a mess...

![image](https://user-images.githubusercontent.com/1549219/183676296-cd750b1c-62e4-4145-bd8d-01503e8bc1a2.png)

I manually altered the dproj file to get my newer ones down the bottom and the DTPs/RTPs all bunched together (which is better than the mess you get by default)

The above could be improved of course

Reply:
Do you mean other project group?

Reply:
No, more tree nodes so everything wasn't mixed up as is the case usually

Can't work out how to do that though so just ordered them sensibly - at least I can group select with ease that way...

Reply:
The main Python4Delphi is horrible in this respect. Everything requires Python but it's not the first to be built. After thasn everything wants PythonEngine and that's not in order either.

I'm not certain what order you should build in but invariably just selecting build all ressults in something going wrong - not exactly easy to install...

Reply:
Weird, it should be working fine. Which project group are you using?

Reply:
Compile both for safety (never works first time)

I'll do a full wipe (also harder done than said) then install from scratch and record it all for Video and/or Grabs - it's the only good QA method.

Not doing that right now though - I'm all Delphi'd out ATM - need to switch to anything else for some hours to reset my brain